jsonquery-querydsl-base/src/main/java/com/lindar/jsonquery/querydsl/QuerydslLocalDateCalculator.java

```diff
@@ -0,0 +1,107 @@
+package com.lindar.jsonquery.querydsl;
+
+import com.lindar.jsonquery.ast.DateLocalDateComparisonNode;
+import com.querydsl.core.types.Ops;
+import com.querydsl.core.types.Predicate;
+import com.querydsl.core.types.dsl.DateExpression;
+import com.querydsl.core.types.dsl.Expressions;
+
+import java.time.LocalDate;
+import java.time.temporal.TemporalAdjusters;
+import java.time.temporal.WeekFields;
+import java.util.Locale;
+
+public class QuerydslLocalDateCalculator extends BaseQuerydslDateCalculator<LocalDate, DateLocalDateComparisonNode> {
+
+    public Predicate fromRelative(DateLocalDateComparisonNode dateComparisonNode, DateExpression<LocalDate> dateExpression) {
+
+        switch (dateComparisonNode.getRelativeOperation()) {
+            case IN_THE_LAST:
+                return dateExpression.gt(fromRelativeDate(dateComparisonNode));
```

Comment:
In the previous calculator services you've used `.gte` rather than just `.gt`, pointing out in case its a mistake. 
Actually could be incorrect, "In the last 3 days" would include that 3rd day as well